File: octopoints-flutter/lib/db/entity/UserEntity.dart
import 'package:floor/floor.dart';

@Entity(tableName: 'users')
class UserEntity {
  @PrimaryKey(autoGenerate: true)
  final int? id;
  final String username;
  int win;
  int lose;

  UserEntity(this.username, {this.id, this.win = 0, this.lose = 0});
}

File: octopoints-kotlin/app/src/main/java/dev/tpavani/octopoints/db/entity/UserEntity.kt
package dev.tpavani.octopoints.db.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "users")
class UserEntity(
    @PrimaryKey(autoGenerate = true)
    val id: Int,
    val username: String,
    var win: Int,
    var lose: Int,
)